setup.py

```diff
@@ -212,6 +212,7 @@ def run_setup(
             'shap.plots', 'shap.plots.colors', 'shap.benchmark', 'shap.maskers', 'shap.utils',
             'shap.actions', 'shap.models'
         ],
+        package_data={'shap': ['plots/resources/*', 'cext/tree_shap.h']},
```

Comment:
Change to `'cext/*'` to include the missing files?

Reply:
Agreed, done